sv/message.go

```diff
@@ -80,7 +81,7 @@ func (p MessageProcessorImpl) SkipBranch(branch string, detached bool) bool {
 // Validate commit message.
 func (p MessageProcessorImpl) Validate(message string) error {
 	subject, body := splitCommitMessageContent(message)
-	msg := p.Parse(subject, body)
+	msg, _ := p.Parse(subject, body)
```

Comment:
i think it's better to return the error in this case too, since it could be a config error, it's better to know it soon than later..

what do you think?

Reply:
Probably yes. Will change it